### ISSUE TITLE
Ducky Keyboard doesn't start when using USBHIDBootKbd sketch

### ISSUE THREAD

**Issue by thomaspaulin** [felis/USB_Host_Shield_2.0#738]
I'm using a full-size board on an Arduino Uno and had the board_qc sketch complete successfully. I've moved onto plugging in keyboards and have got my Logitech K270 keyboard working (Bluetooth) using the USBHIDBootKbd sketch without a hitch, however when I plug in my [Ducky One 2 Mini](https://www.duckychannel.com.tw/en/Ducky-One2-Mini-RGB) the keyboard never reports ready and I receive the following debug output:
```
#define ENABLE_UHS_DEBUGGING 1
```

```
09:39:05.843 -> Start
09:39:06.306 -> BM Init
09:39:06.612 -> Addr:01
09:39:06.612 -> NC:01
09:39:06.612 -> HID_PROTOCOL_KEYBOARD
09:39:07.970 -> bNumEP:01
09:39:07.970 -> D1
```

I've read a few issues on this repository which mention N-key rollover as a possible culprit for headaches, something this Ducky indeed includes, but I'd expect that to manifest when pressing keys rather than have it fail to start. Thoughts?

**Comment by thomaspaulin** [comment]
It turns out when the DIP switches are in the default position, the Ducky One 2 Mini isn't a boot device.

**Comment by thomaspaulin** [comment]
It turns out the firmware that shipped with my keyboard was not reporting the device as bootable (in any of the interface descriptors), then specified a keyboard protocol in violation of USB HID device class definitions. 

By updating my firmware to v1.22 the USB_desc sketches shows the keyboard now reports the keyboard is bootable, and lo and behold the USBHIDBootKbd sketch detects key presses.

TL;DR: Try updating your keyboard's firmware to v1.22.
